protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -411,6 +412,7 @@ func renderMessageAsDefinition(msg *descriptor.Message, reg *descriptor.Registry
 		schema.MinProperties = protoSchema.MinProperties
 		schema.Required = protoSchema.Required
 		schema.XNullable = protoSchema.XNullable
+		schema.extensions = protoSchema.extensions
```

Comment:
Would it be worth checking that the key of the extensions start with `x-` as specified in the swagger spec: https://swagger.io/docs/specification/2-0/swagger-extensions/?

Reply:
Good point, fortunately this is already handled in the [`processExtensions` ](https://github.com/grpc-ecosystem/grpc-gateway/blob/b5cd496c648a3394eacb65d9e2f6f07adb2449df/protoc-gen-openapiv2/internal/genopenapi/template.go#L1538) function called a bit further up.